ROS 2: erase superfluous semicolon

Concerning the ROS2 branch

I have a problem when compiling object files using the pluginlib, when I get the following error:
`/include/pluginlib/class_list_macros.hpp:49:59: error: extra ‘;’ [-Werror=pedantic]
   CLASS_LOADER_REGISTER_CLASS(class_type, base_class_type);`

It's claiming that one should remove the last ";" in this line: 
https://github.com/ros/pluginlib/blob/b456beba88564b746694dde68696d8bc4edc78c9/pluginlib/include/pluginlib/class_list_macros.hpp#L49
That makes sense, since macros shouldn't end with a semicolon and indeed, everything still seems to work fine without it and I don't get the above error.

I sadly can't provide a PR at the moment.

--- a/pluginlib/include/pluginlib/class_list_macros.hpp
+++ b/pluginlib/include/pluginlib/class_list_macros.hpp
@@ -46,6 +46,6 @@
  * \param base_class_type The real base class type from which class_type inherits
  */
 #define PLUGINLIB_EXPORT_CLASS(class_type, base_class_type) \
-  CLASS_LOADER_REGISTER_CLASS(class_type, base_class_type);
+  CLASS_LOADER_REGISTER_CLASS(class_type, base_class_type)
 
 #endif  // PLUGINLIB__CLASS_LIST_MACROS_HPP_

